samples/matter/lock/boards/nrf54h20dk_nrf54h20_cpuapp.overlay

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+/ {
+	chosen {
+		/* prepare IPC to HCI Radio core */
+		zephyr,bt-hci-ipc = &ipc0;
+	};
+
+	aliases {
+		factory-data = &factory_data;
```

Comment:
It is currently not used, right?

Reply:
Yes, but I would like to leave it here and repair it ASAP.